src/App/dtml/copyright.dtml

```diff
@@ -9,13 +9,13 @@
   <link rel="stylesheet" type="text/css" href="&dtml-BASEPATH1;/++resource++zmi/bootstrap-4.1.1/bootstrap.min.css" />
```

Comment:
Not really a problem of this PR, but this is actually points to a non-existing bootstrap version.  So let's fix it.

```suggestion
  <link rel="stylesheet" type="text/css" href="&dtml-BASEPATH1;/++resource++zmi/bootstrap-4.6.0/bootstrap.min.css" />
```

Reply:
Thanks, I added this change in the same PR